ansible/roles/ocp4-workload-ai-ml-workflows-demo/tasks/per_user_remove_workload.yml

```diff
@@ -0,0 +1,40 @@
+---
+# Implement your Workload deployment tasks here
+
+- debug:
+    msg: "Removing ML Workflows workshop materials from preexisting project {{ project_name }}"
+
+- name: Create the custom notebook image
+  k8s:
+    state: absent
+    namespace: "{{ project_name }}"
+    definition: "{{ item }}"
+  with_items:
+    - "{{ lookup('template', 'custom-notebook-imagestream.yaml.j2') }}"
```

Comment:
Please switch from `with_items` to `loop`. Also, the output in the logs from doing the lookup here is going to be very hard to read because it is going to dump a json blob in the log. Better to move the lookup into the `definition`:

```
- name: Create the custom notebook image
  k8s:
    state: absent
    namespace: "{{ project_name }}"
    definition: "{{ lookup('template', item) }}"
  loop:
    - custom-notebook-imagestream.yaml.j2
```